lib/fizzy/execute.hpp

```diff
@@ -18,11 +18,14 @@
 
 namespace fizzy
 {
-// The result of an execution.
+/// The result of an execution.
 struct ExecutionResult
 {
+    /// This is true if the execution has trapped. No other fields are valid.
```

Comment:
I suggest to remove this, as Rust part relies on `has_value` being valid in this case now.
```suggestion
    /// This is true if the execution has trapped.
```